### PR TITLE
Hello Microverse project Pull Request.

### PR DESCRIPTION
**Pull request by sinruidotec**
**In this project I:**

- Added an HTML file with h1 and a class.
- Added a CSS file to stylize the h1 class.
- Added .gitignore file with node_modules and test.md files inside.
- Added linters.yml.

HTML and CSS project, the main task is to add the necessary files and folders to code with good practices.

### CODE REVIEW

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @sinruidotec, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good PR description
- Meaningful commit messages
- Linters check have passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly, I suggest you adding your `Readme.md` for review.
- According to the project requirements, Kindly I suggest you adding the `test.md` file with the text `This file should be ignored by git`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,162 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Microverse README Template</b></h3>
```
Comment (Alusp) [inline README.md]: - [x] Good job in implementing the changes requested by the previous reviewer, :clap: , However, Please replace the word: `Microverse README Template` with `Hello Microverse` as the title given in the requirement.

File: README.md [hunk 2]
```diff
@@ -1 +1,162 @@
...
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [] <a name="about-project"></a>
```
Comment (Alusp) [inline README.md]: - [x] Please replace this square bracket: `[]` with `Hello Microverse` as the preferred way according to the requirement.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# test.md
```
Comment (Alusp) [inline .gitignore]: - [x] Please just remove this `#` before the `test.md` word in your gitignore file, by so doing, your test.md file will only stay in your local files and will not be pushed into your git repository as given in the project requirement. :slightly_smiling_face:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @sinruidotec, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

close

## Highlights

- [x] Linters are passing correctly :green_circle: 
- [ ] Good PR summary :green_circle: 
- [x] Meaningful commit messages :green_circle: 
- [x] Professional Readme Template Used :green_circle: 
- [x] Correct GitHub Flow Used :green_circle: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @sinruidotec, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**